### DISCUSSION TITLE
Is there a way for using 'services google sheets sheet import' for adding data from CSV file?

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [watermint/toolbox#700] [Q&A] [answered]
I'm looking for a tool for implement very simple scenario - to add one row from CSV file to existing Google sheet (the same fields and number of fields). Can toolbox do that?

TIA, Vitaly

**Comment by watermint** [comment, marked as answer]
Hi @vitalyk-multinarity.
Thank you for writing in.
Here is the command to append rows into your existing spreadsheet.

https://toolbox.watermint.org/commands/services-google-sheets-sheet-append.html
Hope this helps

**Reply by vitalyk-multinarity** [reply]
Many thanks, somehow I missed it.
It works, thank you again for fantastic tool!
